install-operator.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+OPERATORS_NAMESPACE="openshift-operators"
+
+set -ex
+
+# Create a subscription
+cat <<EOF | oc create -f -
```

Comment:
@e-minguez why not asking users to just do a `oc create -f` instead of running a shell script that does just that?

Reply:
I just wanted to have something to deploy it via the CLI instead using the UI.
Do you want me to rewrite it to avoid running the script and provide CLI instructions?